plugins/auth-backend/config.d.ts

```diff
@@ -51,7 +51,7 @@ export interface Config {
         cert?: string;
         privateKey?: string;
         decryptionPvk?: string;
-        signatureAlgorithm?: 'sha1' | 'sha256' | 'sha512';
+        signatureAlgorithm?: 'sha256' | 'sha512';
```

Comment:
Let me just double check - is this strictly necessary, AFTER actually upgrading? I am not sure if anybody actually uses sha1, but if they do, and usage is safe in v2, maybe we should keep it

Reply:
It is reported in their changelog. https://github.com/node-saml/passport-saml/releases/tag/v2.0.0
Who wan to use sha1 ...

Reply:
Yeah agreed :) But you know, legacy situations or whatnot

Reply:
I know people will complain after upgrade it do not work !

but if you upgrade your install you can pay the price to update the configuration because of security issue.

As you wish 🤷